modules/nf-core/crabz/compress/environment.yml

```diff
@@ -0,0 +1,9 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/environment-schema.json
+name: "crabz_compress"
```

Comment:
```suggestion
name: crabz_compress
```

Reply:
Is this in the documentation? I used `nf-core modules create`.